blivet/devices/storage.py

```diff
@@ -243,7 +243,7 @@ def _setTargetSize(self, newsize):
         if not isinstance(newsize, Size):
             raise ValueError("new size must of type Size")
 
-        if self.maxSize and newsize > self.maxSize:
+        if self.maxSize and newsize and newsize > self.maxSize:
```

Comment:
This happens when the (last) thin LV is removed and the thin pool's size is adapted as part of ``_reconfigure_container()``. The calculated size for thin pool is ``0`` because it has no thin LVs and it now doesn't require any padding.